src/signing.c

```diff
@@ -101,6 +108,10 @@ void aws_crt_signing_config_aws_set_expiration_in_seconds(
     signing_config->config.expiration_in_seconds = expiration_in_seconds;
 }
 
+void aws_crt_signing_config_aws_set_date(aws_crt_signing_config_aws *signing_config, uint64_t timestamp) {
+    aws_date_time_init_epoch_secs(&signing_config->config.date, (double)timestamp);
```

Comment:
same, we blindly init on set without freeing any previous value

Reply:
There's no clean up for `aws_date_time`